base_delivery_carrier_files/wizard/generate_carrier_files.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Guewen Baconnier
+#    Copyright 2012 Camptocamp SA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, exceptions, api
+from openerp.tools.translate import _
+
+
+class DeliveryCarrierFileGenerate(models.TransientModel):
+
+    _name = 'delivery.carrier.file.generate'
+
+    @api.multi
+    def _get_picking_ids(self):
+        context = self.env.context or {}
```

Comment:
`or {}` should not be necessary when getting `self.env.context` this is done by method `__new__()` of env which create a frozen dict.